web/requirements.txt

```diff
@@ -29,3 +29,5 @@ drf-yasg[validation]
 swagger-spec-validator
 flex
 lxml
+pytest-django
+python-dateutil
```

Comment:
Shouldn't you include version numbers? pip freeze --local

Reply:
I did a straight pip freeze (not running in virtualenv, I develop in Docker) and added a requirements-frozen.txt with version numbers that is used in the Dockerfile to install. I checked that the thing still builds and the tests still run